content/release-10-16-0/platform-10-16-0-bundle/10-16-0-0.md

```diff
@@ -185,12 +163,40 @@ MTM-48529</td>
 <td>MTM-48344</td>
 </tr>
 
+<tr>
+<td>REST API</td>
+<td>The performance of the Alarm API has been improved for requests in which the resolved query parameter is used.</td>
+<td>MTM-49450</td>
+</tr>
+
 <tr>
 <td>Security</td>
 <td>An attribute was added to all external application links, that prevents tab nabbing, a security vulnerability that could be used to gain the user's browser session.</td>
 <td>MTM-48156</td>
 </tr>
 
+<td>
+Single sign-on</td>
+<td>Trial tenants created in Software AG Cloud are by default not allowed to access the single sign-on configuration in Cumulocity IoT to prevent its potential misconfiguration by users. A REST endpoint is used by the Management tenant to restrict or allow the access to the configuration for specific tenants. Refer to <a href="https://cumulocity.com/guides/10.16.0/users-guide/administration/#configuration-access">Administration > Single sign-on > Configuration access</a> in the <i>User guide</i> for more details. </td>
+<td>
+MTM-32508</td>
+</tr>
+
+<tr>
+<td>Single sign-on</td>
+<td>The single sign-on (SSO) configuration page has been extended by a new access mapping option. Instead of a checkbox which sets the mapping during each login or only during user creation, a radio button has been added. Now three options are available:
+<br>- Perform access mapping for every login for all privileges (admin can not edit SSO user privileges)

```

Comment:
listings like these are without a period? looking at it, i'm not sure what we decided

Reply:
Good question. @dianaolives could you please help?

Reply:
If at least one of the items on the list is a complete sentence, they must all have period at the end.